Updating terminology Community Connector > Marketplace Connector, Certified Connector > Integration (#40678)
airbytehq/airbyte: docs/contributing-to-airbyte/README.md

## Code Contributions

Most of the issues that are open for contributions are tagged with [`good-first-issue`](https://github.com/airbytehq/airbyte/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue) or [`help-welcome`](https://github.com/airbytehq/airbyte/labels/help-welcome).

> A great place to start looking will be our GitHub projects for [**Marketplace Connector Issues**](https://github.com/orgs/airbytehq/projects/50)

Reviewer: ```suggestion
```

This link is to a project where we have _some_ but not all relevant issues, and a LOT of issues are in the closed repository. Until @marcosmarxm and I fix it, let's remove this link. I feel pretty strongly about this.

> Contributions for new connectors should come with documentation. Refer to the [guidelines on updating documentation](writing-docs.md) which includes a template to use while documenting a new connector. 

Reviewer: ```suggestion
Please include documentation when contributing a new connector. Refer to the [guidelines on updating documentation](writing-docs.md) which includes a template to use while documenting a new connector.
```

@marcosmarxm, one thing that changes with Builder Contribution flow is that we are going to auto-generate documentation for connectors based on their manifest, so separate docs are no longer going to be a thing. But for now, this stands.
